[2024-06-28T14:19:38.246+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-28T14:19:38.320+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_sample_dag.task_from_script1 scheduled__2023-08-16T05:15:00+00:00 [queued]>
[2024-06-28T14:19:38.359+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_sample_dag.task_from_script1 scheduled__2023-08-16T05:15:00+00:00 [queued]>
[2024-06-28T14:19:38.365+0000] {taskinstance.py:2306} INFO - Starting attempt 2 of 2
[2024-06-28T14:19:38.403+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): task_from_script1> on 2023-08-16 05:15:00+00:00
[2024-06-28T14:19:38.424+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=2678) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-28T14:19:38.429+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'my_sample_dag', 'task_from_script1', 'scheduled__2023-08-16T05:15:00+00:00', '--job-id', '1910', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/tmp/tmp1w7ap_sd']
[2024-06-28T14:19:38.433+0000] {standard_task_runner.py:91} INFO - Job 1910: Subtask task_from_script1
[2024-06-28T14:19:38.429+0000] {standard_task_runner.py:63} INFO - Started process 2788 to run task
[2024-06-28T14:19:38.546+0000] {task_command.py:426} INFO - Running <TaskInstance: my_sample_dag.task_from_script1 scheduled__2023-08-16T05:15:00+00:00 [running]> on host 81089ac3a69d
[2024-06-28T14:19:38.732+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='my_sample_dag' AIRFLOW_CTX_TASK_ID='task_from_script1' AIRFLOW_CTX_EXECUTION_DATE='2023-08-16T05:15:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-16T05:15:00+00:00'
[2024-06-28T14:19:38.737+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-28T14:19:38.740+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-28T14:19:38.741+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/script1.py", line 10, in function_from_script1
    response = requests.get(url)
                            ^^^
NameError: name 'url' is not defined
[2024-06-28T14:19:38.761+0000] {taskinstance.py:1206} INFO - Marking task as FAILED. dag_id=my_sample_dag, task_id=task_from_script1, run_id=scheduled__2023-08-16T05:15:00+00:00, execution_date=20230816T051500, start_date=20240628T141938, end_date=20240628T141938
[2024-06-28T14:19:38.786+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 1910 for task task_from_script1 (name 'url' is not defined; 2788)
[2024-06-28T14:19:38.840+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-06-28T14:19:38.857+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
